RepairsListView

vanyalaptii opened the pull request:
RepairsListView

vanyalaptii committed:
Fixed bug and working on RepairsListView

Fixed bug with xcodeproj file, working on RepairsListView

vanyalaptii committed:
Строка листу, NavigationTitle, Toolbar

OksanaFedorchuk commented on ReFix/Views/RepairsListView.swift:
```diff
@@ -12,26 +12,56 @@ struct RelpairsListView: View {
+                    .padding()
+                    .font(.largeTitle)
+                }
+                
```
зайві пусті строки

vanyalaptii replied:
Виправив

OksanaFedorchuk commented on ReFix/Views/RepairsListView.swift:
```diff
@@ -12,26 +12,56 @@ struct RelpairsListView: View {
+                
+                
         }
+        
```
зайві пусті строки

vanyalaptii replied:
Виправив

OksanaFedorchuk commented on ReFix/Views/RepairsListView.swift:
```diff
@@ -12,26 +12,56 @@ struct RelpairsListView: View {
+                            
+                            Spacer()
+                            
+                            HStack {
```
давай починати виносити деякі елементи в проперті, наприклад ячейку тут

private extension RelpairsListView {
      var listRow(_ item: Item): some View  {
             //code
       }
}

vanyalaptii replied:
Я не зовсім зрозумів
Можливо щось не так роблю
Але якщо я намагаюся робити так як вказано вище то компілятор свариться на: (_ item: Item)

OksanaFedorchuk replied:
я просто приклад написала)) пиши своє

vanyalaptii replied:
Зробив)

OksanaFedorchuk commented on ReFix/Views/RepairsListView.swift:
```diff
@@ -12,26 +12,56 @@ struct RelpairsListView: View {
+                            }
+                        }
+                        .padding(.horizontal)
+                        
```
зайві строки

vanyalaptii replied:
Виправив

vanyalaptii committed:
Пошук

Додав пошук. Є можливість пошуку по бренду, моделі, імені клієнту та номеру телефону, також по номеру ремонту.

vanyalaptii committed:
Виніс єлемент у extension

vanyalaptii committed:
Fixed some details
